benchmarks/pca/datasets/simulated_blobs.py

```diff
@@ -0,0 +1,30 @@
+from benchopt import BaseDataset, safe_import_context
+from benchopt.datasets import make_correlated_data
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+
+
+class Dataset(BaseDataset):
+    name = "Simulated_correlated_data"
+
+    parameters = {
+        "n_samples, n_features": [(1_000_000, 10_000), (10_000, 10_000)],
+        "dtype": ["float32", "float64"],
```

Comment:
I would rather just run the benchmark on `float32` data which all libraries should support (and that is probably enough in practice).